lint check fails after upgrading google-java-format 1.15.0

The shared config 1.3.1 had google-java-format 1.15.0 (2022-03-07)

The shared [config 1.3.0](https://github.com/googleapis/java-shared-config/blob/v1.3.0/pom.xml#L228) had google-java-format 1.7.0 (2019-01-09).

https://github.com/googleapis/java-shared-config/blob/v1.3.0/pom.xml#L228


Initial report https://github.com/googleapis/java-bigquerystorage/pull/1590 by @stephaniewang526 

It's failing in other places https://github.com/googleapis/java-retail/pull/379#issuecomment-1080777342 :

```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Google Cloud Retail Parent 2.0.17-SNAPSHOT:
[INFO] 
[INFO] Google Cloud Retail Parent ......................... SUCCESS [  0.650 s]
[INFO] proto-google-cloud-retail-v2 ....................... FAILURE [  0.776 s]
[INFO] grpc-google-cloud-retail-v2 ........................ SKIPPED
[INFO] Google Cloud Retail ................................ SKIPPED
[INFO] Google Cloud Retail BOM ............................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.078 s
[INFO] Finished at: 2022-03-26T17:42:56Z
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal com.coveo:fmt-maven-plugin:2.9:check (default-cli) on project proto-google-cloud-retail-v2: Execution default-cli of goal com.coveo:fmt-maven-plugin:2.9:check failed: A required class was missing while executing com.coveo:fmt-maven-plugin:2.9:check: com/google/googlejavaformat/java/RemoveUnusedImports$JavadocOnlyImports
Error:  -----------------------------------------------------
Error:  realm =    plugin>com.coveo:fmt-maven-plugin:2.9
Error:  strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
Error:  urls[0] = file:/home/runner/.m2/repository/com/coveo/fmt-maven-plugin/2.9/fmt-maven-plugin-2.9.jar
Error:  urls[1] = file:/home/runner/.m2/repository/com/google/googlejavaformat/google-java-format/1.15.0/google-java-format-1.15.0.jar
Error:  urls[2] = file:/home/runner/.m2/repository/com/google/guava/guava/31.0.1-jre/guava-31.0.1-jre.jar
Error:  urls[3] = file:/home/runner/.m2/repository/com/google/guava/failureaccess/1.0.1/failureaccess-1.0.1.jar
Error:  urls[4] = file:/home/runner/.m2/repository/com/google/guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar
Error:  urls[5] = file:/home/runner/.m2/repository/com/google/code/findbugs/jsr305/3.0.2/jsr305-3.0.2.jar
Error:  urls[6] = file:/home/runner/.m2/repository/org/checkerframework/checker-qual/3.12.0/checker-qual-3.12.0.jar
Error:  urls[7] = file:/home/runner/.m2/repository/com/google/errorprone/error_prone_annotations/2.7.1/error_prone_annotations-2.7.1.jar
Error:  urls[8] = file:/home/runner/.m2/repository/com/google/j2objc/j2objc-annotations/1.3/j2objc-annotations-1.3.jar
Error:  urls[9] = file:/home/runner/.m2/repository/org/codehaus/plexus/plexus-utils/3.0.22/plexus-utils-3.0.22.jar
Error:  urls[10] = file:/home/runner/.m2/repository/org/apache/commons/commons-lang3/3.4/commons-lang3-3.4.jar
Error:  urls[11] = file:/home/runner/.m2/repository/javax/enterprise/cdi-api/1.0/cdi-api-1.0.jar
Error:  urls[12] = file:/home/runner/.m2/repository/javax/annotation/jsr250-api/1.0/jsr250-api-1.0.jar
Error:  urls[13] = file:/home/runner/.m2/repository/org/eclipse/sisu/org.eclipse.sisu.inject/0.3.2/org.eclipse.sisu.inject-0.3.2.jar
Error:  urls[14] = file:/home/runner/.m2/repository/org/codehaus/plexus/plexus-component-annotations/1.5.5/plexus-component-annotations-1.5.5.jar
Error:  Number of foreign imports: 1
Error:  import: Entry[import  from realm ClassRealm[project>com.google.cloud:google-cloud-retail-parent:2.0.17-SNAPSHOT, parent: ClassRealm[maven.api, parent: null]]]
Error:  
Error:  -----------------------------------------------------: com.google.googlejavaformat.java.RemoveUnusedImports$JavadocOnlyImports
Error:  -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginContainerException
Error:  
Error:  After correcting the problems, you can resume the build with the command
Error:    mvn <args> -rf :proto-google-cloud-retail-v2
basename: missing operand
Try 'basename --help' for more information.
exiting with 1
Error: Process completed with exit code 1.
```

`A required class was missing while executing com.coveo:fmt-maven-plugin:2.9:check: com/google/googlejavaformat/java/RemoveUnusedImports$JavadocOnlyImports` implies `com/google/googlejavaformat/java/RemoveUnusedImports$JavadocOnlyImports` is missing.

--- a/pom.xml
+++ b/pom.xml
@@ -225,7 +225,7 @@
             <dependency>
               <groupId>com.google.googlejavaformat</groupId>
               <artifactId>google-java-format</artifactId>
-              <version>1.15.0</version>
+              <version>1.7</version>
             </dependency>
           </dependencies>
         </plugin>

